neps/nep-0001.md

```diff
@@ -68,13 +68,39 @@ The NEP process begins when an author submits a [NEP draft](#submit-a-nep-draft)
 
 ![NEP Process](https://user-images.githubusercontent.com/110252255/201413632-f72743d6-593e-4747-9409-f56bc38de17b.png)
 
-
 ### NEP Stages
 
-- **Draft:** The first formally tracked stage of a new NEP. This process begins once an author submits a draft proposal and the NEP moderator merges it into the NEP repo when properly formatted. 
+- **Draft:** The first formally tracked stage of a new NEP. This process begins once an author submits a draft proposal and the NEP moderator merges it into the NEP repo when properly formatted.
 - **Review:** A NEP moderator marks a NEP as ready for Subject Matter Experts Review. If the NEP is not approved within two months, it is automatically rejected.
 - **Voting:** This is the final voting period for a NEP. The working group will vote on whether to accept or reject the NEP. This period is limited to two weeks. If during this period necessary normative changes are required, the NEP will revert to Review.
 
+Moderator, when moving a NEP to review stage, should update the Pull Request description to include the
+review summary, example:
+
+```markdown
+---
+
+## NEP Status _(Updated by NEP moderators)_
+
+SME reviews:
+
+- [ ] Role1: @github-handle
+- [ ] Role2: @github-handle
+
+Contract Standards WG voting indications (❔ | 👍 | :-1: ):
+
+- ❔ @frol
+- ❔ @mfornet
+- ❔ @abacabadabacaba
+- ❔ @fadeevab
+- ❔ @robert-zaremba
```

Comment:
not sure if in a template we should list the names here. It makes it easier to copy paste.

Reply:
These handles only represent Contract Standards WG members, so the copy paste argument is not justified. Let's replace it with a placeholder as it would be cumbersome to update the template every time we add a new WG and invite/remove WG members.

Reply:
OK

Reply:
removed names from the template